src/components/Menu/Item.js

```diff
@@ -0,0 +1,142 @@
+import { css } from '@emotion/react'
+import PropTypes from 'prop-types'
+import React from 'react'
+import Button from '../Button'
+import Link from '../Link'
+
+const styles = {
+  borderless: css`
+    border: 0;
+  `,
+  danger: theme => css`
+    color: ${theme.colors.red};
+    display: inline-block;
+    &:hover,
+    &:focus {
+      color: ${theme.colors.darkRed};
+      svg {
```

Comment:
Just a thought, not related to this PR, but we might need to handle the Icon hover better in the future

Either to add the logic to add an Icon/Name/Left component inside this component or move the css hover selector where we use this component.

But in either way this component should not style elements that might not be down the tree